Vagrantfile

```diff
@@ -158,6 +158,7 @@ Vagrant.configure('2') do |config|
       cfg.vm.provision :chef_client do |chef|
         chef_defaults chef, symbol
         chef.add_recipe 'cerner_splunk::cluster_slave'
+        chef.add_recipe 'cerner_splunk_test::_configure_guids'
```

Comment:
Let's have this commented out, and a note in the Vagrant docs about how to wire up the cluster to the established pools. That way we don't have to start a license master with the cluster.